Project Files/Calculator/Calculator.java

```diff
@@ -110,55 +110,11 @@ public void setPool_volume(int pool_volume) {
     }
 
     public void setPool_volume(int length, int width, int height, String shape) {
-        pool_volume = calc_pool_volume(length, width,height, shape);
+        pool_volume = calcPoolVolume(length, width,height, shape);
     }
 
-    //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
I was originally thinking we might be able to use static methods for some of the calculations, but after getting more familiar with the way the calculations need to be done I see that this wont be possible.